packages/teleport/src/services/kube/makeKube.ts

```diff
@@ -17,7 +17,8 @@ limitations under the License.
 import { Kube } from './types';
 
 export default function makeKube(json): Kube {
-  const { name, labels = [] } = json;
+  const { name } = json;
+  const labels = json.labels ? json.labels : [];
```

Comment:
```suggestion
  const labels = json.labels || [];
```

simplified. as stated earlier, `undefined` and `null` are falsey values, so if false, empty array will be assigned